isabelle/test/Spec/Core/Serialization/golden/dolar-token.golden

```diff
@@ -0,0 +1 @@
+{"currency_symbol":"85bb65","token_name":"dolar"}
```

Comment:
Tests like this that use a currency symbol that is invalid in Cardano will fail if we use them in `marlowe-cardano`. Should we fill out these out to 28 bytes?

Reply:
For the JSON specification a dummy value should be fined, for the test oracle, I already noted that these tests should be parametric on an Address and Token type (related to [PR 135](https://github.com/input-output-hk/marlowe/pull/135)).

An address/token  that works for a cardano implementation might not work for an eventual Ethereum implementation. That is why in the initial skeleton for the `test oracle client` I included a [Generate Random Type command](https://github.com/input-output-hk/marlowe-cardano/pull/194/files#diff-28027e9d0c42d8150e1b516cda019f0daceeedbd18601ef7c190f6531add347bR44). The idea would be for that the driver requests the client for some random `tokens` and `addresses` and then use that in other serialization/computeTransaction tests.

Reply:
Having said that, I am thinking of adding a `Cardano` theory with some constants like `adaToken`, `djedToken` and any relevant information that could be used when defining cardano contracts in Isabelle.